assets/scripts/script.js

```diff
@@ -48,14 +48,12 @@
           selector: '.gallerya__image > a'
         });
       }
-      if ($('body').hasClass('woocommerce') && $('.images').length > 0) {
-        $('.images').lightGallery({
-          thumbnail: true,
-          showThumbByDefault: false,
-          subHtmlSelectorRelative: true,
-          selector: '.zoom'
-        });
-      }
+      $('.woocommerce-product-gallery__image').lightGallery({
```

Comment:
The wrapping container for the product gallery is `.woocommerce-product-gallery`

There is also `.woocommerce-product-gallery--with-images` and we should check in which cases that class is set – probably if there are any images in the gallery (as opposed to none) – perhaps the update announcement blog post explains it as well?

Reply:
`.woocommerce-product-gallery--with-images` is not in the post, but it is set (conditionally) only in one place:

https://github.com/woocommerce/woocommerce/blob/103f674d185c6aff3ed18c9762825675f61d3151/templates/single-product/product-image.php#L28.

It looks like a rather specific selector, as it clearly indicates 'a product gallery with images'. I will use it then.

Pushed a new fixup with this correction.